pkg/controllermanager/controller/add.go

```diff
@@ -65,6 +68,24 @@ func AddControllersToManager(mgr manager.Manager, cfg *config.ControllerManagerC
 		return fmt.Errorf("failed adding Quota controller: %w", err)
 	}
 
+	restConfig, err := kubernetes.RESTConfigFromClientConnectionConfiguration(&cfg.GardenClientConnection, nil, kubernetes.AuthTokenFile)
+	if err != nil {
+		return err
+	}
```

Comment:
Can you use the REST config created in `app.go` instead of duplicating it? This adds the risk of being missed on important changes.